### ISSUE TITLE
Electroniques dans un appartement meublé

### ISSUE THREAD

**Issue by Ninie08** [datagir/nosgestesclimat#1927]
Bonjour, 

Je viens de faire le test, et pour le logement, j'aurais aimé savoir si ça fait une différence quand on habite dans un meublé où les appareils électroménagers ou autre n'ont pas été achetés par moi-même et qui serviront surement à moi ou à d'autres locataires pendant de nombreuses années. 

Merci!

**Comment by Ninie08** [comment]
Et aussi, pourquoi est-ce que l'item "privilégier le train à l'avion" reduirait mon empreinte de 16%, mais qu'arrêter complètement de prendre l'avion la réduirait de 15% seulement?

**Comment by JuliePouliquen** [comment]
Bonjour @Ninie08 !

Merci beaucoup pour tes deux questions. Pour la première, c'est indirectement pris en compte aujourd'hui vu que nous comptons au niveau de NGC : les achats neufs pour la partie construction (donc ici a priori vous n'en avez pas, cela se joue au niveau du propriétaire), la consommation d'électricité pour la partie usage (bien compris dans votre facture).
Pour ce qui est de votre deuxième question... c'est bien étrange et ça mérite investigation ! Je me permets de rajouter @Clemog avec qui je vais regarder cela.

Bonne journée !

**Comment by Clemog** [comment]
> * Arrêter l'avion supprime tout simplement totalement l'empreinte liée aux transports en avion. Les trajets sont effacés, il n'y a pas de compensation par un autre moyen de transport : `- transport . avion`
> * Privilégier le train à l'avion remplace l'impact des vols court-courrier par l'empreinte des mêmes km effectués en train, avec un facteur multiplicateur de 1,2 pour symboliser le nombre de km supérieur réalisé par rapport à la ligne droite dans les airs (plus ou moins) en avion : `- transport . avion . court courrier + compensation train`

Exactement, à mon avis on est dans le cas d'une personne dont les vols sont uniquement des vols courts-courrier : ils sont remplacés par le train (ce n'est pas le cas pour l'action arrêter l'avion) et donc on a une légère différence.

**Comment by Clemog** [comment]
Oups il me semble que je me suis trompé .. @Ninie08 pouvez vous nous envoyer les données que vous avez entré dans le test au niveau de l'avion pour que nous puissions reproduire le bug ?

**Comment by Clemog** [comment]
> Merci beaucoup pour tes deux questions. Pour la première, c'est indirectement pris en compte aujourd'hui vu que nous comptons au niveau de NGC : les achats neufs pour la partie construction (donc ici a priori vous n'en avez pas, cela se joue au niveau du propriétaire), la consommation d'électricité pour la partie usage (bien compris dans votre facture).

Je me permets de compléter : les émissions des appareils de votre logement vous seront attribuée, même si c'est votre propriétaire qui est acheteur. On demande en effet pour le test, la composition du logement pour des appareils de moins de 10 ans.

**Comment by JuliePouliquen** [comment]
Oups merci @Clemog pour le rectificatif.

**Comment by lekda** [comment]
Bonjour, 
Même problème que Ninie08, remplacer l'avion par le train est une action qui d'après le simulateur me permet d'éviter plus de kg de CO2 que juste arrêter l'avion. 
En tout état de cause ça devrait être l'inverse, remplacer l'avion par le train devrait permettre d'économiser l'avion - l'impact du train et pas l'avion + l'impact du train. 
Quand on regarde le publicode associé, on trouve un `+` à la place d'un `-` pour la compensation train : peut-être juste un signe à changer ? 

![image](https://github.com/incubateur-ademe/nosgestesclimat/assets/26326375/40bdf5cd-3e60-4e6b-88f1-75b99845ed18)
![image](https://github.com/incubateur-ademe/nosgestesclimat/assets/26326375/c66c15bd-0df5-49c8-bf71-07478f9eb1ff)
![image](https://github.com/incubateur-ademe/nosgestesclimat/assets/26326375/300b9699-6b28-4fd8-9110-428b5f6fef4d)
